sys/openbsd/dev_bpf_amd64.const

```diff
@@ -0,0 +1,29 @@
+# AUTOGENERATED FILE
+# AT_FDCWD is not set
```

Comment:
This needs some other header. Not critical as I hope we have this in other const files, but "is not set" in itself is kind of a red flag. Keeping each file self-contained may be useful for future if we do static verification or something.